.github/workflows/build.yml

```diff
@@ -0,0 +1,146 @@
+name: Prebuild
+
+on:
+  push:
+    # tags:
+    #   - v*
+
+env:
+  NAPI_VERSION: 3
+  BINDING_NAME: HID
+  BINDING_NAME_RAW: HID_hidraw
```

Comment:
This would be better as `BINDING_NAME_HIDRAW` instead of `BINDING_NAME_RAW`, as its the `hidraw` backend that this alternate version is using